website/templates/search_bar.mako

```diff
@@ -14,3 +15,31 @@
         </div>
     </div>
 </div>
+
+<script>
+/** Only for Search Bar Placeholder -- Allow IE and other browsers to work as the same */
```

Comment:
@haoyuchen1992 We try not to put javascript into the view files, there might still be some examples in the OSF but we are trying to move away from that. This can probably go into "navbarControl.js".